lib/node/nodes/georeference-street-address.js

```diff
@@ -49,42 +53,55 @@ GeoreferenceStreetAddress.prototype.getGeocoderParams = function () {
     return geocoderParams;
 };
 
-var COLUMNS_TEMPLATE_REGEX = /\{\{([^\}]*)\}\}/g;
+var COLUMNS_TEMPLATE_REGEX = /\{\{([^\}]*)\}\}/;
 GeoreferenceStreetAddress.prototype.getStreetAddressColumnParam = function() {
     if (this.street_address_column) {
         return this.getFunctionParam('street_address');
     }
 
-    var columns = [];
-    var result = this.street_address_template.replace(COLUMNS_TEMPLATE_REGEX, function(s, column) {
-        columns.push(column.trim());
-        return '%s';
+    var columns_and_literals = this.street_address_template.trim().split(', ').map(function(fragment) {
```

Comment:
This is a bit complex to understand.

Why do we split by `', '`?

Reply:
I guess tests were "_happy_" use-cases, but the previous code didn't assume any format as it can be any free text coming from the user. For instance, a user could input `{{street_number}} {{street_name}}. {{postal_code}} {{city}}`.

Reply:
Ok, I'll improve that, taking into account that I can't just split by spaces in order to support things like `{{street_number}} {{street_name}}, La Rioja`.